src/busctl/busctl.c

```diff
@@ -1477,7 +1478,7 @@ static int status(int argc, char **argv, void *userdata) {
         return 0;
 }
 
-static int message_append_cmdline(sd_bus_message *m, const char *signature, char ***x) {
+static int message_append_cmdline(sd_bus_message *m, const char *signature, FDSet **fdset, char ***x) {
```

Comment:
hmm, maybe rename fdset to passed_fdset? just to make clear what this is about, i.e. the fds we got passed in